lib/const.rb

```diff
@@ -50,7 +50,7 @@
 # These are packages that crew needs to run-- only packages that the bin/crew needs should be required here.
 # lz4, for example, is required for zstd to have lz4 support, but this is not required to run bin/crew.
 # The LIBC_VERSION ternary is to reflect the change from glibc_build to glibc_lib as the source of essential libraries starting at glibc 2.35.
-CREW_ESSENTIAL_PACKAGES ||= %W[gcc_lib #{LIBC_VERSION.to_f > 2.34 ? "glibc_lib#{LIBC_VERSION.delete('.')}" : "glibc_build#{LIBC_VERSION.delete('.')}"} glibc_standalone gmp ruby zlib zlib_ng zstd]
+CREW_ESSENTIAL_PACKAGES ||= %W[crew_profile_base gcc_lib #{LIBC_VERSION.to_f > 2.34 ? "glibc_lib#{LIBC_VERSION.delete('.')}" : "glibc_build#{LIBC_VERSION.delete('.')}"} glibc_standalone gmp ruby zlib zlib_ng zstd]
```

Comment:
Aren't we removing the glibc_build# and glibc_lib# packages in https://github.com/chromebrew/chromebrew/pull/11911?

Reply:
Yes, but this PR will need to be merged first before #11911. And then remove `glibc_*` in #11911 afterwards.

Reply:
Yes and WHEN that happens... 😅